adapted-articles/J/JesusChristTeachingsof.xml

```diff
@@ -14,77 +14,77 @@
+<p>One of the most characteristic titles Jesus used for God was Father. This was not new, because the idea occurs in the OT, where God is seen as Father of his people Israel. This kind of fatherhood was national rather than personal. In the period between the old and new testaments the Jews came to regard God as so holy that he was removed from direct contact with human affairs. There had to be mediators between God and people. This elevated notion of God did not lead to the idea of God as Father. It is against this background that we must see Jesus’s unique teaching of the personal fatherhood of God. There is some evidence in Judaism that prayer to God as “Our Father” was known, but what distinguishes Jesus’s teachings from that of his contemporaries is that the fatherhood of God was central to what he taught.</p>
+<p>The father-son relationship is particularly vivid in John’s Gospel, where Jesus as Son is seen to be in close communion with God as Father. This comes out strongly in Jesus’s prayer in <a href="?bref=John.17.1-26">John 17</a> and in the frequent assertions that the Father had sent the Son and that the Son was doing the will of the Father. It is this strong relationship between God and Jesus in terms of fatherhood and sonship that led Jesus to teach men to approach God in the same way. The Lord’s Prayer at once recognizes this in its opening words. It is particularly important to note that “Our Father” precedes “may your name be kept holy” (<a href="?bref=Matt.6.9">Mt 6:9</a>), because the it is a more intimate idea and prepares the way for the more remote. Jesus never taught men to approach God with terror.</p>
+<p>Although there is a connection between the way in which Jesus addressed God as Father and the way in which he taught his disciples to approach God, there is also a distinction. Jesus spoke of “my Father and your Father” when he appeared to Mary Magdalene after his resurrection (<a href="?bref=John.20.17">Jn 20:17</a>), but he did not say “our Father.” His sonship was unique, because he claimed that he and the Father were one (<a href="?bref=John.10.30">Jn 10:30</a>).</p>
+<p>In the Sermon on the Mount, Jesus assured his followers that their heavenly Father knows about their needs (<a href="?bref=Matt.6.32">Mt 6:32</a>; <a href="?bref=Luke.12.30">Lk 12:30</a>). On this strong basis they are encouraged not to be anxious. This gives some insight into the way in which Jesus’s teachings about God has a bearing on practical issues.</p>
```

Comment:
On this basis we are strongly encouraged...

Reply:
Well, I can do that if you like, but grammatically its correct, since he talks about “the followers” in the previous sentence.

Reply:
It's "this strong basis" that sounds weird to me.

Reply:
I guess the author meant that it is a “firm foundation,” that Jesus’s followers should be encouraged by it and not allow themselves to be anxious.

In that sense, it seems to me that dropping the “strong” won’t harm the general understanding of the article.

Reply:
We could even go further and simplify it as such:

> In the Sermon on the Mount, Jesus assured his followers that their heavenly Father knows about their needs (Mt 6:32; Lk 12:30), and shouldn’t be anxious.

But, I'm thinking on refraining on such simplification, unless it is convincing to you.